influxql/statement_rewriter.go

```diff
@@ -1,6 +1,8 @@
 package influxql
 
-import "errors"
+import (
```

Comment:
This should be formatted with `goimports`. I think `goimports` defaults a single import to being a single line instead of using parenthesis.